src/3.1/entrypoint.sh

```diff
@@ -15,40 +15,32 @@ export LD_LIBRARY_PATH=${MUSIC_PATH}/lib:$LD_LIBRARY_PATH
 export PATH=${MUSIC_PATH}/bin:$PATH
 export CPATH=${MUSIC_PATH}/include:$CPATH
 export PYTHONPATH=${MUSIC_PATH}/lib/python3.8/site-packages:$PYTHONPATH
-export NEST_SERVER_RESTRICTION_OFF=true
-export NEST_SERVER_MODULES=nest,numpy
 
-if [[ ! -d /opt/data ]]; then
-    mkdir /opt/data
-fi
 
-if [[ "$1" = 'notebook' ]]; then
-    cd /opt/data
-    exec jupyter-notebook --ip="${IP_ADDRESS}" --port=8080 --no-browser --allow-root
-fi
+MODE="${NEST_CONTAINER_MODE:-$1}"
+if [[ "${MODE}" = 'interactive' ]]; then
+    read -p "Your python script: " name
+    echo Starting: $name
+    # Start
+    mkdir -p /opt/data; cd /opt/data
+    exec python3 /opt/data/$name
 
-if [[ "$1" = 'jupyterlab' ]]; then
-    cd /opt/data
+elif [[ "${MODE}" = 'jupyterlab' ]]; then
+    mkdir -p /opt/data; cd /opt/data
     exec /root/.local/bin/jupyter-lab --ip="${IP_ADDRESS}" --port=8080 --no-browser --allow-root
-fi
 
-if [[ "$1" = 'nest-server' ]]; then
-    cd /opt/data
-    exec nest-server start -o -h 0.0.0.0 -p 5000 -u 65534
-fi
+elif [[ "${MODE}" = 'nest-desktop' ]]; then
+    exec /root/.local/bin/nest-desktop start -h 0.0.0.0 -p 8000
 
-if [[ "$1" = 'nest-desktop' ]]; then
-    cd /opt/data
-    exec /root/.local/bin/nest-desktop start
-fi
+elif [[ "${MODE}" = 'nest-server' ]]; then
+    export NEST_SERVER_RESTRICTION_OFF=true
+    export NEST_SERVER_MODULES=nest,numpy
```

Comment:
If you use the same bash default, then one could also e.g. add modules from the outside. (This is optional and no reason to reject this PR)
```suggestion
    export NEST_SERVER_RESTRICTION_OFF=${NEST_SERVER_RESTRICTION_OFF:-true}
    export NEST_SERVER_MODULES=${NEST_SERVER_MODULES:-nest,numpy}
```